.flake8

```diff
@@ -48,6 +48,8 @@ extend-ignore =
 # https://wemake-python-stylegui.de/en/latest/pages/usage/formatter.html
 #format = wemake
 
+inline-quotes = "
```

Comment:
This is coming from a third-party plugin and all of such settings are maintained at the end of the file.